lua/core/autocmds.lua

```diff
@@ -0,0 +1,112 @@
+local autocommands = {}
```

Comment:
what is this variable used for?

Reply:
It's used to return the `define_augroups` function that we need to call after `lvim.autocommands` has been updated by the user. For now it's called from `init.lua`